### PR TITLE
[DPE-3514]  Add Apache Kyuubi entrypoint to Charmed Spark base rock image

### PR DESCRIPTION
**Pull request by theoctober19th**
This PR adds Apache Kyuubi entrypoint to the base image of Charmed Spark. 
Jira:
https://warthogs.atlassian.net/browse/DPE-3514

### CODE REVIEW

**Review by welpaolo — DISMISSED** [review]
Looks good to me

File: rockcraft.yaml [hunk 1]
```diff
@@ -95,9 +93,41 @@ parts:
...
+      - opt/kyuubi/charts
+      - opt/kyuubi/conf
+      - opt/kyuubi/db-scripts
+      - opt/kyuubi/docker
+      - opt/kyuubi/externals
+      - opt/kyuubi/jars
+      - opt/kyuubi/logs
+      - opt/kyuubi/pid
+      - opt/kyuubi/web-ui
+      - opt/kyuubi/work
```
Comment (deusebio) [inline rockcraft.yaml]: **niptick, question** why not just organizing and staging `/opt/kyuubi`? Are there other forders we don't want to copy over?

**Review by deusebio — DISMISSED** [review]
Just a small non-blocking comment. All good otherwise. Please only add the ticket number in the PR description

File: rockcraft.yaml (hunk 1 above)
Comment (theoctober19th) [reply rockcraft.yaml]: The following files / folders are not copied over:
```
embedded_zookeeper/ 
kyuubi_state_store.db
LICENSE
licenses/
NOTICE
RELEASE
```

There is no particular reason for leaving behind this files. We could as well stage entire `/opt/kyuubi`.

Comment (theoctober19th) [reply rockcraft.yaml]: Updated to dump the entire directory structure.

**Review by welpaolo — APPROVED** [review]
LGTM :)

**Review by deusebio — APPROVED** [review]

**Pull request merged**